doc/docs/master/deploy-manage/deploy/deploy-pachyderm-jupyterhub.md

```diff
@@ -0,0 +1,245 @@
+### Enable Pachyderm Access Controls (Optional)
+
+You can enable Pachyderm access controls to use Pachyderm credentials
+to log in to JupyterHub. However, this
+is not required, and if you prefer to use JupyterHub authentication
+instead, skip this section.
+
+Pachyderm Access Controls is an enterprise feature. You need to have
+an active Enterprise token to enable it.
+
+To enable Pachyderm authentication, follow the steps in
+[Configure Access Controls](https://docs.pachyderm.com/latest/enterprise/auth/auth/)
+
+!!! note
+    If you decide to enable Pachyderm authentication later, you
+    will need to either manually reconfigure or redeploy JupyterHub.
+
+### Install Helm
+
+Helm is a package manager for Kubernetes that controls the JupyterHub
+installation.
+
+To install Helm on your client machine, follow the steps in the
+[Helm Documentation](https://helm.sh/docs/intro/install/).
+
+## Deploy JupyterHub
+
+After you deploy Pachyderm in one of the supported environments and
+install Helm on your client machine, you can use the deployment script
+to deploy JupyterHub.
+
+To deploy JupyterHub, complete the following steps:
+
+1. Clone the [jupyterhub-pachyderm](https://github.com/pachyderm/jupyterhub-pachyderm)
+repository:
+
+   ```bash
+   git clone git@github.com:pachyderm/jupyterhub-pachyderm.git
+   ```
+
+1. Change the current directory to `jupyterhub-pachyderm`:
+
+   ```bash
+   cd jupyterhub-pachyderm
+   ```
+
+1. Run the `./init.py` script:
+
+   ```bash
+   ./init.py
+   ```
+
+   **System response:**
+
+   ```bash
+   ===> checking dependencies are installed
+   ...
+   ===> installing jupyterhub
+   Release "jhub" does not exist. Installing it now.
+   NAME: jhub
+   LAST DEPLOYED: Wed Dec 18 13:09:26 2019
+   NAMESPACE: default
+   STATUS: deployed
+   REVISION: 1
+   TEST SUITE: None
+   NOTES:
+   Thank you for installing JupyterHub!
+   ...
+   ===> notes
+   - Since Pachyderm auth doesn't appear to be enabled, JupyterHub will expect the following global password for users: XXXXXXXXXXXXXXXXXXXXX
+   ```
+
+   The JupyterHub is now deployed on your Kubernetes cluster.
+
+## Log in to JupyterHub
+
+After you deploy JupyterHub, you can access the JupyterHub UI
+in a web browser through the Pachyderm cluster hostname on port
+`80`.
+
+If you have enabled Pachyderm access controls, use your Pachyderm
+token to log in. Otherwise, use the global password
+that was printed out when you ran the `./init.py` script with any
+username.
+
+## Using `python-pachyderm`
+
+After you log in, you can use the [Python Pachyderm](https://pachyderm.github.io/python-pachyderm/python_pachyderm.m.html#header-functions)
+client API to manage Pachyderm directly from your Jupyter notebooks.
+
+The following lines initialize the Python Pachyderm client in JupyterHub:
+
+```bash
+import python_pachyderm
+client = python_pachyderm.Client.new_in_cluster()
+```
+
+!!! note
+    This syntax is different from the one you run directly
+    from your terminal.
```

Comment:
1) nit: the syntax is the same, but the function call is different.
2) I could see this statement confusing some users, since you can run a terminal in JupyterHub. Instead, I think it should say it's different from the function you'd call locally.